core/src/main/resources/application.yml

```diff
@@ -792,9 +792,11 @@ cb:
       - name: cm
         pkg:
           - name: cloudera-manager-agent
-            pattern: "(.*)-(.*)\\..*"
+            pattern: "(.*)-([0-9]+)[a-zA-z]*\\..*"
+            validateForMultipleVersions: true
```

Comment:
Wouldn't object creation from config fail if the new field is added here but the PackageName class is not extended with the new field? Or is it just ignored?